Contents/Code/siteRealityLovers.py

```diff
@@ -0,0 +1,121 @@
+    data = urllib.urlopen(req).read()
+    searchResults = json.loads(data)
+
+    for searchResult in searchResults['contents']:
+        releaseDate = parse(searchResult['released']).strftime('%b %d %Y')
+        curID = searchResult['videoUri'].replace('/', '_').replace('?', '!')
+        score = 100 - Util.LevenshteinDistance(searchTitle.lower(), title.lower())
+        siteName = PAsearchSites.getSearchSiteName(siteNum)
+        titleNoFormatting = '%s [%s] %s' % (searchResult['title'], siteName, releaseDate)
+        results.Append(MetadataSearchResult(id='%s|%d' % (curID, siteNum), name=titleNoFormatting, score=score, lang=lang))
+
+    return results
+
+
+def update(metadata, siteID, movieGenres, movieActors):
+    Log('******UPDATE CALLED*******')
+
+    id = str(metadata.id).split('|')
+    videoUri = id[0].replace('_', '/')
+    url = PAsearchSites.getSearchBaseURL(siteID) + videoUri
+    detailsPageElements = HTML.ElementFromURL(url)
+
+    metadata.collections.clear()
+    movieGenres.clearGenres()
+    movieActors.clearActors()
+
+    # Studio
+    metadata.studio = PAsearchSites.getSearchSiteName(siteID)
+
+    # Title
+    metadata.title = detailsPageElements.xpath('//h1[@class="video-detail-name"]')[0].text_content().strip()
+
+    # Summary
+    rawSummary = detailsPageElements.xpath('//p[@itemprop="description"]')[0].text_content()
+    formattedSummary = re.sub(' +', ' ', rawSummary.replace("…", "").replace("Read more", "").replace("\n", ""))
+    metadata.summary = formattedSummary
+
+    # Tagline and Collection(s)
+    metadata.tagline = metadata.studio
+    metadata.collections.add(metadata.studio)
+
+    # Genres
+    genres = detailsPageElements.xpath('//span[@itemprop="keywords"]/a')
+    if len(genres) > 0:
+        for genreLink in genres:
+            genreName = genreLink.text_content().strip().lower()
+            movieGenres.addGenre(genreName)
+
+    # Release Date
+    date = detailsPageElements.xpath('//span[@class="videoClip__Details-infoValue"]')[0].text_content().strip()
+    if len(date) > 0:
+        unformattedDate = parse(date).strftime('%b %d %Y')
+        date_object = datetime.strptime(unformattedDate, '%b %d %Y')
+        metadata.originally_available_at = date_object
+        metadata.year = metadata.originally_available_at.year
+
+    # Actors
+    actors = detailsPageElements.xpath('//span[@itemprop="actors"]/a')
+    if len(actors) > 0:
+        for actorLink in actors:
+            actorName = str(actorLink.text_content().strip())
+            try:
+                actorPageURL = actorLink.get("href")
+                actorPage = HTML.ElementFromURL(actorPageURL)
+                actorPhotoLinks = actorPage.xpath('//img[@class="girlDetails-posterImage"]')[0].get("srcset")
+                actorPhotoURLs = str(actorPhotoLinks).split(',')
+                actorPhotoURL = actorPhotoURLs[1][:-3].replace("https", "http")
```

Comment:
Couldn't figure out how to make urllib2 verify the SSL cert. If anyone knows please let me know. In trying to fix it I learned the Plex app module using urllib2 has had this issue for sometime. Err when requesting the secure site is: `CERTIFICATION_VERIFY_FAILED`.

Reply:
Try to set `PYTHONHTTPSVERIFY` environment variable to 0

Reply:
This didn't work for me. I tried `export PYTHONHTTPSVERIFY=0` and got the same err. Some other way I should be setting it? I assume it needs to be global because the err is coming from the py env in the Plex app?

Reply:
export works only for current shell session, for global try to edit `/etc/environment` or something like that.